### PR TITLE
5.65

### PR DESCRIPTION
**Pull request by seamuslee001**
@eileenmcnaughton does this look correct to you?

### CODE REVIEW

File: CRM/Core/EntityTokens.php [hunk 1]
```diff
@@ -152,6 +152,7 @@ public function evaluateToken(TokenRow $row, $entity, $field, $prefetch = NULL)
    * @return bool
    */
   public function isHTMLTextField(string $fieldName): bool {
+    $metadata = $this->getMetadataForField($fieldName);
```
Comment (larssandergreen) [inline CRM/Core/EntityTokens.php:155]: Don't think we actually need this

Comment (seamuslee001) [reply CRM/Core/EntityTokens.php:155]: its in the 5.65 version but maybe not?

Comment (eileenmcnaughton) [reply CRM/Core/EntityTokens.php:155]: it's a mistake - let's merge this & fix after

Comment (larssandergreen) [reply CRM/Core/EntityTokens.php:155]: I was doing a little follow up on event tokens and included removing this in there
https://github.com/civicrm/civicrm-core/pull/27349

**Pull request merged**